functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectConfigurator.java

```diff
@@ -58,9 +61,12 @@ public ProjectConfigurator withConfiguration(Configuration configuration) {
         return this;
     }
 
-    public ProjectConfigurator createConfigFile() {
+    public ProjectConfigurator createConfigFile(String... customConfigFile) {
```

Comment:
if we are always taking the first one, why not creating two methods `createConfigFile` one with no args and the other with just one string